extension/src/experiments/model/filterBy/index.test.ts

```diff
@@ -0,0 +1,271 @@
+import { FilterDefinition, filterExperiment, Operator } from '.'
```

Comment:
[F] This file got purged in https://github.com/iterative/vscode-dvc/pull/3585. I brought it back/modified it to bring back some coverage.